data/faq.html

```diff
@@ -1062,6 +1272,29 @@ <h3 lang="es">Búsqueda</h3>
 <p></p>
 Nota: solamente puedes encontrar los sitios que han sido mapeados en 
 <a href="http://www.openstreetmap.org">OpenStreetMap</a>.</dd>
+<dd lang="fr">Cliquer sur l'icone de recherche dans l'écran principal. Vous pouvez rechercher en
+utilisant l'un des cas suivants :
+<p></p>
+<b><em>Adresse</em></b> : entrez le N° de la maison, le nom de la rue. Vous pouvez également inquer
+le nom de la ville (si vous ne trouvez toujours pas) et le code postal.
+Par exemple: 41, Rue des Alouettes 75000 Paris.
```

Comment:
Bad example: the search returns nothing for this address.
1. It doesn't look like we have the postal code info for this point:
http://www.openstreetmap.org/node/3290564280
2. I'm not even sure 75000 is the correct one for this particular house.